app/models/concerns/orchestration.rb

```diff
@@ -119,6 +119,13 @@ def skip_orchestration_for_testing?
     Rails.env.test?
   end
 
+  def without_orchestration_if(disabled = true, &block)
+    skip_orchestration! if disabled && SETTINGS[:unattended]
```

Comment:
Looks like the setting check here is useless as this concern is only included if `SETTINGS[:unattended]==true`: https://github.com/theforeman/foreman/blob/develop/app/models/host/managed.rb#L293-L297
Causing fact import to fail when it is set to false - https://projects.theforeman.org/issues/31364